### ISSUE TITLE
bug: black screen

### ISSUE THREAD

**Issue by jakub-beep** [ionic-team/ionic-framework#25588]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [X] v5.x
- [ ] v6.x
- [ ] Nightly

### Current Behavior

ionic/react : “5.6.8”
ionic/react-router: “5.6.9”
react: “17.0.2”
next: “11.0.1”
Hi, I am facing issues with black screen and components losing its styling during interaction with a page. It happens usually when I am clicking on the buttons which call some popups which are ionic components. I am pretty sure it is also connected with routing. This little css property is called during those events. I can of course block this css in the code, but it is pointless because when I set it to transparent I see that components lost its styling, also I cannot interact with the page. I see that Ionic 6 probably resolves those issues, because after migration I did not face any issues, but I cannot be sure, because many things just crashed and I would have to upgrade them to test it fully. Unfortunately it is quite impossible to migrate our app to ionic 6 right now because there is no enough time and there is no guarantee that everything will be fine.
Is there anyone who faced those issues and could help me to solve this problem? Please help.

### Expected Behavior

When rerouting I expect not to lose styling and not to see blackscreen popup.

### Steps to Reproduce

Honestly I am not sure.

### Ionic Info

ionic/react : “5.6.8”
ionic/react-router: “5.6.9”

**Comment by Sukaato** [comment]
I have the same issue, when i go to a page that use an `ion-datetime` i got a blackscreen (depends on theme) and some errors:
```
TypeError: Failed to resolve module specifier "@utils/logging". Relative references must start with either "/", "./", or "../". undefined
```
```
Uncaught (in promise) Error: Constructor for "ion-datetime#undefined" was not found
    at initializeComponent (index-163bbb8a.js:2040:23)
```

**Comment by sean-perkins** [comment]
@Sukaato I have an open PR, but here is a dev-build in the interim: `6.1.14-dev.11657236641.106931ae`

**Comment by liamdebeasi** [comment]
Thanks for the issue! This issue is being closed due to the lack of a code reproduction. If this is still an issue with the latest version of Ionic, please create a new issue and ensure the template is fully filled out.

The separate issue mentioned in https://github.com/ionic-team/ionic-framework/issues/25588#issuecomment-1178192415 has been resolved in Ionic 6.1.14.

Thank you for using Ionic!
